.snyk

```diff
@@ -6,4 +6,9 @@ ignore:
     - pdfmake:
```

Comment:
missing unit test for the complete output of a schedule when it is a top-up, a reduction and 0 value for `app/generator/content/index.js`

Reply:
as above.